src/test/spec/v2_runner/mod.rs

```diff
@@ -0,0 +1,304 @@
+pub mod operation;
```

Comment:
This (and the rest of the v2 test runner) is just moved from the previous location in the CRUD spec tests, correct?

Reply:
yep, I just did some renaming for the sake of clarity